packages/aws/data_stream/cloudtrail/elasticsearch/ingest_pipeline/default.yml

```
@@ -51,13 +51,14 @@ processors:
   - script:
       description: Appends any relevant entity to `related.entity` for all events
       lang: painless
+      if: "ctx?.json?.eventName != null && ctx?.tags != null && ctx.tags.contains('actor_target_mapping')"
       on_failure: 
         - set:
             description: Add error reason
             field: error.message
             value: "{{{ _ingest.on_failure_message }}}"
```

Please add more details to the error.

```suggestion
            value: "{{{ _ingest.on_failure_message }}}"
            value: >-
              Processor '{{{ _ingest.on_failure_processor_type }}}'
              {{#_ingest.on_failure_processor_tag}}with tag '{{{ _ingest.on_failure_processor_tag }}}'
              {{/_ingest.on_failure_processor_tag}}failed with message '{{{ _ingest.on_failure_message }}}'
```